extensions/wikia/Tasks/proxy/proxy.php

```diff
@@ -34,6 +34,11 @@
 
 $command = "{$env}php {$script} --wiki_id={$wikiId} --task_id={$taskId} --task_list={$list} --call_order={$order} --created_by={$createdBy} --created_at={$createdAt}";
 
+// @todo (PLATFORM-2759): For a long-term fix, initialize the MW stack here, remove the shell_exec and execute the
+//                        TaskRunnerMaintenance directly. For now I'm removing the lines below as the composer/autoload.php
+//                        fails to load some classes without MW and the whole script crashed on devboxes.
+/*
+ *
 // can't use globals here, this doesn't execute within mediawiki
 if ( getenv( 'WIKIA_ENVIRONMENT' ) == 'dev' ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)